.github/workflows/win.yml

```diff
@@ -0,0 +1,45 @@
+name: windows
+
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        #CONDA_PY: ["36", "37"]
+        CONDA_PY: ["37"]
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        auto-activate-base: false
+        miniconda-version: '4.8.3'
```

Comment:
```suggestion
        miniconda-version: '4.8.3'
        conda-version: '4.8.3'
```

Reply:
Any reason to pin to a specific conda version? I would use the latest for this CI.